mindmeld/models/taggers/taggers.py

```diff
@@ -277,6 +278,8 @@ def get_entities_from_tags(query, tags):
 
     entities = []
 
+    sys_resolver = sys_resolver or DucklingRecognizer.get_instance()
```

Comment:
please remove these calls.

Reply:
this is needed here since it is an independent function